src/main/java/com/whitedeath/organizationchart/Controller/EmployeeController.java

```diff
@@ -0,0 +1,179 @@
+package com.whitedeath.organizationchart.Controller;
+
+import com.fasterxml.jackson.annotation.JsonView;
+import com.whitedeath.organizationchart.Model.Designation;
+import com.whitedeath.organizationchart.Model.Employee;
+import com.whitedeath.organizationchart.Model.PutEmployee;
+import com.whitedeath.organizationchart.Repository.DesignationRepo;
+import com.whitedeath.organizationchart.Repository.EmployeeRepo;
+import com.whitedeath.organizationchart.View.Views;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+
+@RestController
+public class EmployeeController {
+
+    private final Comparator<Employee> employeeComparator;
+    private final EmployeeRepo employeeRepo;
+    private final DesignationRepo designationRepo;
+
+    public EmployeeController(EmployeeRepo repo, DesignationRepo drepo) {
+        super();
```

Comment:
why do we have a super call here? I don't see this class extending anything

Reply:
i just mistakenly wrote that. but i have removed that line now.